src/components/footer/_footer.scss

```diff
@@ -49,4 +49,8 @@
       margin-bottom: 0.5rem !important;
     }
   }
+
+  &__poweredByLink {
+    display: contents;
+  }
```

Comment:
What was the thinking of using `display: contents` here? I wasn't sure if it was adding anything